cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -3028,12 +3028,77 @@
 
   <property>
     <name>app.program.runtime.monitor.topics.configs</name>
-    <value>audit.topic,data.event.topic,metadata.messaging.topic,metrics.topic.prefix:${metrics.messaging.topic.num},program.status.event.topic,program.status.event.topic:${program.status.event.topic.num.partitions},log.tms.topic.prefix:${log.publish.num.partitions}</value>
+    <value>app.delete.event.topic,audit.topic,data.event.topic,metadata.messaging.topic,metrics.topic.prefix:${metrics.messaging.topic.num},program.status.event.topic,program.status.event.topic:${program.status.event.topic.num.partitions},log.tms.topic.prefix:${log.publish.num.partitions}</value>
     <description>
       A comma-separated list of topic config to be monitored by runtime monitor
     </description>
   </property>
 
+  <property>
+    <name>app.delete.event.topic</name>
+    <value>appdeleteevent</value>
+    <description>
+      Topic name used to publish app deletion events to the messaging system.
+    </description>
+  </property>
+
+  <property>
+    <name>app.delete.event.fetch.size</name>
+    <value>100</value>
+    <description>
+      Number of messages to fetch from messaging system for each batch
+    </description>
+  </property>
+
+  <property>
+    <name>app.delete.event.poll.delay.millis</name>
+    <value>2000</value>
+    <description>
+      The delay in milliseconds to checks again for new app delete events after it detects there was no event
+    </description>
+  </property>
+
+  <property>
+    <name>system.appdelete.retry.policy.base.delay.ms</name>
```

Comment:
be consistent with the naming. This should have the same prefix as the topic setting.

`system.appdelete.` -> `app.delete.event.`